src/mumble/API_v_1_x_x.cpp

```diff
@@ -1971,7 +2050,9 @@ MumbleAPI_v_1_0_x getMumbleAPI_v_1_0_x() {
 			 setMumbleSetting_string_v_1_0_x,
 			 sendData_v_1_0_x,
 			 log_v_1_0_x,
-			 playSample_v_1_0_x };
+			 playSample_v_1_0_x,
+			 requestChannelLink_v_1_3_x,
+			 requestChannelUnlink_v_1_3_x };
```

Comment:
These can't appear here as these functions are not part of the interface v1.0

Reply:
I tried to remove these definitions, but the compiler gave an error. Also, you gave me the answer, and I can solve it using `#if` .